server/lib/field_hub/project.ex

```diff
@@ -201,6 +201,12 @@ defmodule FieldHub.Project do
       db_statistics ->
         file_statistics = evaluate_file_store(project_identifier)
 
+        changes = CouchService.get_n_last_changes(project_identifier, nb_changes_to_display)
+
+        db_statistics =
+          db_statistics
+          |> Map.put(:last_n_changes, changes)
+
```

Comment:
This should probably pushed into the `evaluate_database/1` function that got called above.